src/app/clusters/color-control-server/color-control-server.h

```diff
@@ -101,6 +101,7 @@ class ColorControlServer
         uint8_t finalHue;
         uint16_t stepsRemaining;
         uint16_t stepsTotal;
+        uint16_t timeRemaining;
```

Comment:
Please document the meaning of these timeRemaining members.  What are the units?  How are they expected to evolve when each step happens?  

Because I don't understand why we reduce `timeRemaining` by 1 every UPDATE_TIME_MS.

Reply:
I've added a comment to the declaration indicating its purpose and another comment to where it gets updated.